src/script/components/Modals/PrimaryModal/PrimaryModal.test.tsx

```diff
@@ -80,4 +83,51 @@ describe('PrimaryModal', () => {
 
     expect(secondaryActionCallback).toHaveBeenCalledTimes(1);
   });
+
+  it('shows close button by default', async () => {
+    const {container} = render(<PrimaryModalComponent />);
+
+    act(() => {
+      PrimaryModal.show(PrimaryModal.type.CONFIRM, {
+        primaryAction: {
+          action: () => {},
+          text: 'test-text',
+        },
+        secondaryAction: {
+          action: () => {},
+          text: 'secondary-text',
+        },
+        text: {
+          message: 'test-message',
+          title: 'test-title',
+        },
+      });
+    });
+    const closeButton = container.querySelector('button[data-uie-name="do-close"]');
```

Comment:
you can use `getByTestId('do-close')` - we've `data-uie-name` configured as our `testid`